lib/client/api_login_test.go

```diff
@@ -296,110 +295,6 @@ func TestTeleportClient_Login_local(t *testing.T) {
 	}
 }
 
-// TestTeleportClient_PromptMFAChallenge tests logic specific to the
-// TeleportClient's wrapper of PromptMFAChallenge.
-// Actual prompt and login behavior is tested by TestTeleportClient_Login_local.
-func TestTeleportClient_PromptMFAChallenge(t *testing.T) {
```

Comment:
I don't think we need it anymore, but WDYT?

The new version of this test would be to test `tc.NewMFAPrompt` to make sure it sets the prompt fields as expected, but this can be verified by just reading the method. Additionally, tests that rely on the client MFA prompt should depend on this functionality.

Also the previous test was more necessary due to the difficulty of tracking mfa opts and how/when they are applied. I think it's plain to see now.